scout/server/blueprints/cases/templates/cases/individuals_table.html

```diff
@@ -40,17 +40,15 @@
               <td>{{ ind.msi or 'N/A' }}</td>
               {% if ind.phenotype_human == "tumor" %}
                 <td>
-                  <div class="row">
-                  {% if not ind.tumor_purity or ind.tumor_purity|float < 0.2 %}
-                  <div class="col-1">
-                    <div class="badge bg-danger rounded-pill">
-                      <span class="fa fa-exclamation " data-bs-toggle='tooltip' data-bs-container='body' title="Low or missing tumor purity"></span>
-                    </div>
-                  </div>
-                  {% endif %}
-                  <div class="col-11"><input name="tumor_purity.{{ind.individual_id}}" type="number" step="0.01" min="0.1" max=1
-                  class="form-control col-7" value="{{ind.tumor_purity}}"></div>
```

Comment:
it was probably this that was merged on - or forgotten. But the d-flex center is good - lets keep it!